internal_transfer_with_agreed_amount/tests/test_internal_transfer_with_agreed_amount.py

```diff
@@ -2,23 +2,24 @@
 
 
 class TestInternalTransferWithAgreedAmount(TransactionCase):
-    def setUp(self):
```

Comment:
Don't you need to inherit `SavepointCase` for `setUpClass` to work?

Reply:
I use this test as a reference: https://github.com/odoo/odoo/blob/e12eacf9161013935598fb2183a56fa1f8390be0/addons/account/tests/test_chart_template.py#L12-L15

They inherit `TransactionCase` when using `SetUpClass`.

Reply:
That test you're pointing out is in 16.0, on which `SavepointCase` [doesn't exist anymore](https://github.com/odoo/odoo/pull/62031)

But, since this is in 12.0, `SavepointCase` is the correct class to use the method `setUpClass`. Then, when we migrate to 16.0, we will replace `SavepointCase` by `TransactionCase`

Reply:
Got it. Changed to SavepointCase.